proposals/simd/NewOpcodes.md

```diff
@@ -12,8 +12,6 @@
 | v128.load32_splat  | 0x09   |
 | v128.load64_splat  | 0x0a   |
 | v128.store         | 0x0b   |
```

Comment:
Looks like we have 0x0c-0x0f (4 opcodes) free now, we could move `f64x2.{ceil,floor,trunc,nearest}` here so they are not as scattered?

Reply:
No, 0x0c-0x0f are occupied byt const, shuffle, swizzle, and a splat. Were you looking somewhere else?

Reply:
Whoops, misread, forgot to expand the collapsed diffs.